### PR TITLE
New version: OpenMedia.4KVideoDownloaderPlus version 1.4.3.0060

### PR DESCRIPTION
**Pull request by fancybody**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137574)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-02T02:14:48.8334938Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 694.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

Is the 64-bit version packaged with an installer that cannot run on 64-bit OSes?

**Comment by fancybody** [comment]
@stephengillie, please rerun the pipeline.

(for future reference, do only the moderators have the permission to restart the pipeline?)
> @wingetbot run

**Comment by stephengillie** [comment]
Hi @fancybody,

> (for future reference, do only the moderators have the permission to restart the pipeline?)

Yes - but your commit should also have triggered a run. We're working through a bit of an issue currently.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  [1BE0:1984][2024-02-08T18:52:36]w363: Could not create system restore point, error: 0x80070422. Continuing...

(Automated response - build 723.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/o/OpenMedia/4KVideoDownloaderPlus/1.4.3.0060/OpenMedia.4KVideoDownloaderPlus.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  AppsAndFeaturesEntries:
+  - UpgradeCode: '{5B6DD0A5-3535-4E25-80B2-46C406B36E88}'
+- Architecture: x64
+  Scope: machine
+  InstallerType: wix
+  InstallerUrl: https://dl.4kdownload.com/app/4kvideodownloaderplus_1.4.3_x64.msi
+  InstallerSha256: D4EB8A04B41957ABB03F3885E1BDD8E71435B4D9CD36022C1D9532FC13F7BBBB
+  ProductCode: '{6B28A8C5-AA96-45C6-9AA2-E7AE5CC5077F}'
+  AppsAndFeaturesEntries:
+  - UpgradeCode: '{5B6DD0A5-3535-4E25-80B2-46C406B36E88}'
```
Comment (stephengillie) [inline manifests/o/OpenMedia/4KVideoDownloaderPlus/1.4.3.0060/OpenMedia.4KVideoDownloaderPlus.installer.yaml:32]:
```suggestion
- Architecture: x86
  InstallerType: burn
  InstallerUrl: https://dl.4kdownload.com/app/4kvideodownloaderplus_1.4.3_online.exe
  InstallerSha256: ed15841b7e073238e895e4f82173f1b7ec0a62837ab0127ac693728cf718862a
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
- Architecture: x64
  InstallerType: burn
  InstallerUrl: https://dl.4kdownload.com/app/4kvideodownloaderplus_1.4.3_x64_online.exe
  InstallerSha256: c25bed34c9abfb4842dd16142e516aeefd4b101d05a681d7d4fefa249a7dc58d
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: x86
  Scope: machine
  InstallerType: wix
  InstallerUrl: https://dl.4kdownload.com/app/4kvideodownloaderplus_1.4.3.msi
  InstallerSha256: 6721EBA32FF864E738661B98C804D3128CFAFE197CD2155D83D2DBB3EC26636A
  ProductCode: '{987010A5-0E88-41EB-AC59-530B8104FB38}'
  AppsAndFeaturesEntries:
  - UpgradeCode: '{5B6DD0A5-3535-4E25-80B2-46C406B36E88}'
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
- Architecture: x64
  Scope: machine
  InstallerType: wix
  InstallerUrl: https://dl.4kdownload.com/app/4kvideodownloaderplus_1.4.3_x64.msi
  InstallerSha256: D4EB8A04B41957ABB03F3885E1BDD8E71435B4D9CD36022C1D9532FC13F7BBBB
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  ProductCode: '{6B28A8C5-AA96-45C6-9AA2-E7AE5CC5077F}'
  AppsAndFeaturesEntries:
  - UpgradeCode: '{5B6DD0A5-3535-4E25-80B2-46C406B36E88}'
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**